<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaRegistro</class>
 <widget class="QDialog" name="PantallaRegistro">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de Usuario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Formulario de Registro</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNickname">
     <property name="text">
      <string>Nombre de Usuario:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUsuario">
     <property name="placeholderText">
      <string>Introduce tu nombre de usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelContrasena">
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditContrasena">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Introduce tu contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelConfirmarContrasena">
     <property name="text">
      <string>Confirmar Contraseña:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditConfirmarContrasena">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirma tu contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCorreo">
     <property name="text">
      <string>Correo Electrónico:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditCorreo">
     <property name="placeholderText">
      <string>Introduce tu correo electrónico</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelConfirmarCorreo">
     <property name="text">
      <string>Confirmar Correo Electrónico:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditConfirmarCorreo">
     <property name="placeholderText">
      <string>Confirma tu correo electrónico</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFechaNacimiento">
     <property name="text">
      <string>Fecha de Nacimiento:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEditNacimiento">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAvatar">
     <property name="text">
      <string>Avatar (opcional):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditAvatar">
     <property name="placeholderText">
      <string>Introduce la ruta de tu avatar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botonSeleccionarAvatar">
     <property name="text">
      <string>Seleccionar Avatar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PantallaRegistro</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
